### PR TITLE
Java verification work for Lesson 2

### PR DESCRIPTION
**Pull request by AnzhelaKhamitskaya**
Name and type of variable

### CODE REVIEW

**Review by AleksandrBohan — APPROVED** [review]
lgfm

File: homework/java/anzhela_khamitskaya/lecture2/tasks.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+oпределение типа переменной является важной задачей, поскольку тип должен отражать ту сущность, которую нужно хранить,
+а также обозначает набор операций над переменной. Например, при работе с математическими операциями лучше выбирать 
+численный тип данных вместо строк, даже если в строке содержатся числа, поскольку над строкой математические операции
+выполнить нельзя.
+ 
+## Требования:
+укажите тип для следующих переменных (можно указать все подходящие варианты)
+
+##
+Количество кораблей в порту - bite
```
Comment (DenisAbramovDm) [inline homework/java/anzhela_khamitskaya/lecture2/tasks.md:41]: Why nof int?

**Review by DenisAbramovDm — APPROVED** [review]
Lgfm

**Review by steleur — APPROVED** [review]

File: homework/java/anzhela_khamitskaya/lecture2/tasks.md (hunk 1 above)
Comment (AnzhelaKhamitskaya) [reply homework/java/anzhela_khamitskaya/lecture2/tasks.md:41]: > Why nof int?
I offer 127 ships for the port - too much. So I've choice byte which type takes less memory (and have made a grammatical mistake). But most likely you are right: in real projects use int.

Comment (GardDen) [reply homework/java/anzhela_khamitskaya/lecture2/tasks.md:41]: > > Why nof int?
> > I offer more than 127 ships for the port - too much. So I've choice byte which type takes less memory (and have made a grammatical mistake). But most likely you are right: in real projects use int.

**Pull request merged**